Packs/IntSight/Integrations/IntSight/IntSight.py

```diff
@@ -1,8 +1,10 @@
 from CommonServerPython import *
 
-reload(sys)
-sys.setdefaultencoding('utf-8')  # pylint: disable=E1101
+# import sys
+import os
+# import importlib
 
+# importlib.reload(sys)
```

Comment:
Commented these lines out as this is a Python 2 integration, and also wasn't sure why they are necessary.
Can you explain or may I delete them?

Reply:
These libraries were included by Demisto back in the days. We tried not to touch anything that was there already. It is completely okay to remove them if that doesn't cause any issue with the overall integration

Reply:
As for the `time.sleep()`:

I looked at the documentation and it looks like the polling has to be implemented on the playbook level (please correct me if I am wrong). Is there a way to implement in on a code level? If yes, then how? Thanks

Reply:
It should actually be implemented at the code level.
You need to create one command that allows for requesting for enrichment, and one command that queries the status and updates the context with it.
Then, we get the results of the enrichment, and update the context.